seldon/seldon-core-operator/base/resources.yaml

```diff
@@ -3666,10 +3656,8 @@ webhooks:
     matchExpressions:
     - key: seldon.io/controller-id
       operator: DoesNotExist
-  objectSelector:
-    matchExpressions:
-    - key: seldon.io/controller-id
-      operator: DoesNotExist
+    - key: serving.kubeflow.org/inferenceservice
+      operator: Exists
```

Comment:
Could we we match specific labels? like "enabled" to match KFServing
https://github.com/kubeflow/kfserving/pull/571/files#diff-2351c76b931b04dcaa68b4d83b8cb372R149

If we can match multiple labels as opposed to just one lets do "enabled" and "true"